db/dcrpg/queries.go

```diff
@@ -1347,37 +1395,74 @@ func retrieveAddressIoCsv(ctx context.Context, db *sql.DB, address string) (csvR
 	return
 }
 
-func scanPartialAddressQueryRows(rows *sql.Rows, addr string) (addressRows []*dbtypes.AddressRow, err error) {
+func scanMergedRows(rows *sql.Rows, addr string, queryType int) (addressRows []*dbtypes.AddressRow, err error) {
 	for rows.Next() {
 		var addr = dbtypes.AddressRow{Address: addr}
 		var blockTime dbtypes.TimeDef
 
-		err = rows.Scan(&addr.TxHash, &addr.ValidMainChain, &blockTime.T,
-			&addr.Value, &addr.MergedDebitCount)
-		addr.TxBlockTime = blockTime
+		var value int64
+		switch queryType {
+		case mergedCreditQuery:
+			err = rows.Scan(&addr.TxHash, &addr.ValidMainChain, &blockTime.T,
+				&value, &addr.MergedCount)
+			addr.IsFunding = true
+		case mergedDebitQuery:
+			err = rows.Scan(&addr.TxHash, &addr.ValidMainChain, &blockTime.T,
+				&value, &addr.MergedCount)
+		case mergedQuery:
+			err = rows.Scan(&addr.TxHash, &addr.ValidMainChain, &blockTime.T,
+				&addr.AtomsCredit, &addr.AtomsDebit, &addr.MergedCount)
+			value = int64(addr.AtomsCredit - addr.AtomsDebit)
+			if value >= 0 {
+				addr.IsFunding = true
+			} else {
+				value = -1 * value
```

Comment:
Is the idea to treat this transaction as a "net" spend/fund for the address?

Reply:
It is, though I definitely had some reservations about this approach. Separate in/out figures would be nice, but I definitely want a sum as well. The address page table is already short on page space for smaller screens, and three columns with 8-point decimal precision take up a lot of width.